lib/report_email_sender.rb

```diff
@@ -0,0 +1,41 @@
+class ReportEmailSender
```

Comment:
Just found the reference to using either Pony or Mail on the SendGrid docs. I guess I don't have much of a preference either way either